version.py

```diff
@@ -1 +1 @@
-__version__ = "1.1.0"
+__version__ = "1.2.0"
```

Comment:
Why this is a major version? Please read https://semver.org. It looks like this should be or 1.1.1 or 1.2.0-rc.1 and after large changes and tests etc it can be released as well.

Reply:
yeah, sounds reasonable